platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygTextView.swift

```diff
@@ -62,6 +62,7 @@ public class WysiwygTextView: UITextView {
         guard content.text != attributedText || content.selection != selectedRange else { return }
 
         performWithoutDelegate {
+            WysiwygPillsFlusher.flush()
```

Comment:
Could you explain the idea of the flusher with a bit more context?

Having it as a singleton, will that mean if we had 2 editors on screen at the same time they could conflict with each other?

Reply:
Yeah indeed it will have issues, I will definitely remove this as a singleton, it should be singularly attach to a `WysiwygComposerViewModel`/`UITextView`. 

I will add more documentation about that as well, but basically this is a fix for a nasty behaviour on iOS, that doesn't remove the views created by `NSTextAttachmentViewProvider` properly. 

Actually, II was also wondering if I can actually build something where the iOS platform code of the library owns the `NSTextAttachmentViewProvider`, so the hosting app just have to worry about providing the view itself, and doesn't have to care about providing things for the flusher. The main issue with that idea, is that if we build something like that the Pills of the Composer can't share the same UTType as the Pills of the Timeline/legacy composer. So I'm not sure what would be the best here 🤔

Reply:
Note: The can't share UTType is due to the fact that registering the view provider class is this code: 

```swift
NSTextAttachment.registerViewProviderClass(WysiwygAttachmentViewProvider.self,
                                           forFileType: WysiwygAttachmentViewProvider.pillUTType)
```

So iOS would use the latest provided Provider for a given UTType whatever the kind of UI element we are using.